### PR TITLE
Separate BSP repos out of platform

### PR DESCRIPTION
**Pull request by doncem**
All BSP repos are still under platform hood. Separating them out

### CODE REVIEW

**Comment by adusumillipraveen** [comment]
:broken_heart:

File: jobdsl/organisations.groovy [hunk 1]
```diff
@@ -14,7 +14,8 @@ List<Map> orgs = [
         [name: 'FinRem', displayName: 'Financial Remedy', regex: 'finrem.*'],
         [name: 'CDM', regex: '\\b(?:document-management-store-app|dm-shared-infrastructure|ccd.*)\\b'],
         [name: 'IAC', regex: 'ia.*'],
-        [name: 'Platform', regex: '(rpe-.*|send-letter.*|draft-store.*|bulk-scan.*|cmc-pdf-service|feature-toggle.*|private-beta-invitation.*|service-auth-provider-app|spring-boot-template|data-extractor|camunda-.*)'],
+        [name: 'BSP', regex: '(send-letter-service|send-letter-performance-tests|bulk-scan.*)'],
```
Comment (adusumillipraveen) [inline jobdsl/organisations.groovy]: Few other configs here, if you want to change : https://github.com/hmcts/cnp-jenkins-config/blob/master/team-config.yml#L104

Comment (doncem) [reply jobdsl/organisations.groovy]: applied changes. didn't touch namespace though

File: team-config.yml [hunk 2]
```diff
@@ -102,10 +102,10 @@ probate:
     contact_channel: "#probate"
     build_notices_channel: "#probate-jenkins"
 bulk-scan:
-  team: "Software Engineering"
+  team: "Bulk Scanning and Printing"
   namespace: "rpe"
```
Comment (adusumillipraveen) [inline team-config.yml]: Do u want to stick to rpe namespace?

Comment (doncem) [reply team-config.yml]: it'll break all our current build will it not?

Comment (adusumillipraveen) [reply team-config.yml]: yes, just needs some one-off manual intervention

**Pull request merged**